libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -16,6 +17,12 @@ AP_Camera_Backend::AP_Camera_Backend(AP_Camera &frontend, AP_Camera_Params &para
 // update - should be called at 50hz
 void AP_Camera_Backend::update()
 {
+    if (_params.options == 1 && AP::dal().get_armed()) {
```

Comment:
```suggestion
    if (_params.options == 1 && AP::arming().is_armed()) {
```
I think you should stash armed state here, so you only send the command on arm or disarm.

Reply:
Yes, I was thinking the same as @IamPete1.  So we **try** to start recording when the vehicle state changes from disarmed to armed.  .. and then we do the opposite when the vehicle is disarmed.  So create a new backend variable called sokmething like, "last_is_armed" and then use that to check if the state has changed.